### DISCUSSION TITLE
The GitHub Enterprise Server 3.15 is now generally available!

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#144554] [Enterprise]
GHES Version 3.15 gives customers enhanced deployment requirements and security controls. Some other highlights include: 

- We have updated root disk size requirements.
- We have also updated minimum server specs recommended to run GHES.
- You can now interact with project status updates using GraphQL and webhooks.
- Custom properties now support new property types: multi select and true/false.
- Code security configurations are now available. 
- Secret scanning push protection is now supported for content upload REST API endpoints – [create a blob](https://docs.github.com/en/enterprise-server@3.15/rest/git/blobs?apiVersion=2022-11-28) and [create or update file contents](https://docs.github.com/en/enterprise-server@3.15/rest/repos/contents?apiVersion=2022-11-28#create-or-update-file-contents). 
- CodeQL‘s support for Swift and Kotlin is now generally available. 
- Organization owners can now grant a user or team access to all of the repositories in their org with a single click.

With some many great new features - which one(s) are unlocking new potential for your organization? 

Please share questions and feedback in the discussion below.

For more details, please visit [the changelog](https://github.blog/changelog/2024-12-03-github-enterprise-server-3-15-is-now-generally-available/).

**Comment by ScriptAutomate** [comment]
@queenofcorgis Are CodeSpaces part of the roadmap for GHES, or is that something GitHub is only intending to have available in the SaaS path?

**Reply by queenofcorgis** [reply]
Thanks for asking @ScriptAutomate, right now they are only available in GitHub Enterprise Cloud, but I've shared your interest with the product manager for GHES.
